api/handlers/convertors.go

```diff
@@ -0,0 +1,27 @@
+package handlers
+
+import (
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/go-swagger/go-swagger/examples/accountmgr/domain"
+	"github.com/go-swagger/go-swagger/examples/accountmgr/gen/models"
+)
+
+func asUserResponse(d *domain.Account) *models.Account {
```

Comment:
why this function name asUserResponse
and where it is getting used?

Reply:
It was not in use so removed it from code now. Please check.